src/pages/library/Datasets.js

```diff
@@ -454,6 +455,7 @@ const Datasets = () => {
     h(DataBrowserPreviewToggler, {
       onChange: value => {
         setCatalogShowing(value)
+        Ajax().Metrics.captureEvent(events.catalogToggle, { enabled: true })
```

Comment:
Do you want to capture `value` in the event as well?

Reply:
Oh! I meant to capture value instead of the constant `true` there. I will fix it!